app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments, foreign_key: 'post_id'
+  has_many :likes, foreign_key: 'post_id'
+  belongs_to :author, class_name: 'User'
+  after_save :update_posts_counter
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
+
+  def fetch_recent_comments
+    comments.order('created_at DESC').last(5)
+  end
```

Comment:
- [ ] In this line, you're ordering the comments by the creation date descending, which orders them so that the newly created ones are at the top and the old ones are at the bottom. Then, you're fetching the last five comments. Kindly edit this line to fetch the last newly added comments instead of fetching the oldest ones, either by removing the `order` or changing the `last` to `limit` to get the top 5, not the last 5, so it will be like:
```
comments.order('created_at DESC').limit(5)
```
OR

```
comments.last(5)
```

Reply:
Thankyou soo much for the suggestion😊